src/btree/bt_ret.c

```diff
@@ -70,102 +70,71 @@ __key_return(WT_CURSOR_BTREE *cbt)
 }
 
 /*
- * __time_pairs_init --
- *     Initialize the time pairs to globally visible.
+ * __read_col_time_window --
+ *     Retrieve the time window from a column store cell.
  */
-static inline void
-__time_pairs_init(WT_TIME_PAIR *start, WT_TIME_PAIR *stop)
+static void
```

Comment:
We could inline the col version since it's local to this function, but the row version is called from `bt_delete.c` so would need to be in a `.i` file for the inlining to work. I'm inclined to leave these as-is, I don't see a lot of benefit to inlining here. Let me know if you feel differently.

Reply:
That's fine for me.